src/shared/destinations.ts

```diff
@@ -196,7 +214,7 @@ const destinationConfigs: DestinationConfig[] = [
       const sourceDirectory = (packageMetadata.repository as JsonObject)
         .directory;
       if (repoUrl && sourceDirectory) {
-        return `${repoUrl}/tree/master/${sourceDirectory}`;
+        return `${repoUrl}/tree/main/${sourceDirectory}`;
```

Comment:
GitHub now redirects any unknown branch to default branch. e.g.

`kachkaev/njt` default branch is `main` in url we use `master`

https://github.com/kachkaev/njt/blob/master/src/types.ts

Reply:
Hmm maybe we’ll need to test a bit more separately. You are right about redirects from `master` to `main` – that works, so we can safely navigate people to `../master/..` and be sure that they won’t get 404. Not sure that it’s true the other way around though. E.g.

https://github.com/prettier/prettier/blob/master/package.json 302
✅ redirects to
https://github.com/prettier/prettier/blob/main/package.json

https://github.com/cypress-io/cypress/blob/main/package.json 404
❌ does not redirect to
https://github.com/cypress-io/cypress/blob/master/package.json

Reply:
I will revert this change.